frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -94,6 +94,20 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
               }
             </Nav>
 
+            <Nav className="mr-auto">
+              {
+                hasRole(currentUser, "ROLE_USER") && (
+                  <NavDropdown title="Earthquakes" id="appnavbar-earthquakes-dropdown" data-testid="appnavbar-earthquakes-dropdown" >
+                    <NavDropdown.Item href="/earthquakes/list" data-testid="appnavbar-earthquakes-list">List</NavDropdown.Item>
+                    {
+                      hasRole(currentUser, "ROLE_ADMIN") && (
+                        <NavDropdown.Item href="/earthquakes/retrieve" data-testid="appnavbar-earthquakes-retrieve">Create</NavDropdown.Item>
+                      )
```

Comment:
What on earth is this horror of a boolean operator. The right operand is just evaluated and dumped in the JSX as an expression if `hasRole` is truthy?

Reply:
it's very cursed, but the starter code seems to condone it